rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinTestUtils.java

```diff
@@ -29,6 +29,18 @@ public static List<BtcECKey> getBtcEcKeysFromSeeds(String[] seeds, boolean sorte
         return keys;
     }
 
+    /**
+     * Returns a BTC address derived from the seed.
+     *
+     * @param seed              the seed to derive the address from
+     * @param networkParameters the network parameters
+     * @return the BTC address
+     */
+    public static Address getBtcAddress(String seed, NetworkParameters networkParameters) {
```

Comment:
This already exists in this class @wilmerrootstock. See `createP2PKHAddress`

Reply:
Oooh, Thanks for sharing!

Reply:
Done!